org.openstream.music.yaml

```diff
@@ -0,0 +1,51 @@
+app-id: org.openstream.music
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+build-options:
+  append-path: /usr/lib/sdk/node18/bin
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+modules:
+  - name: openstreammusic
+    buildsystem: simple
+    build-options:
+      env:
+        XDG_CACHE_HOME: /run/build/openstreammusic/flatpak-node/cache
+        npm_config_cache: /run/build/openstreammusic/flatpak-node/npm-cache
+        npm_config_offline: 'true'
+    build-commands:
+      # Change directory to 'desktop'
+      - rm -r index.html
+      - mv desktop/* .
+      # Install npm dependencies
+      - npm install --offline
+      # Build the app using npx electron-builder
+      - npx electron-builder --linux src --x64
+      # Copy the built files to the Flatpak app directory
+      - cp -a ./build/linux-unpacked /app/main
+      # Install app wrapper
+      - install -Dm755 -t /app/bin/ ./run.sh
+      - install -Dm644 images/logo.png -t /app/share/icons/hicolor/128x128/apps
+      - install -Dm644 images/logo.png -t /app/share/icons/hicolor/48x48/apps
```

Comment:
```suggestion
```

it surely is not 128x128 and 48x48 at the same time.

Reply:
btw the icon shall be installed with a name that matches the app-id. And the .desktop have the proper `Icon` field set as well.